examples/vc-2.0/vc-jwt.jsonld

```diff
@@ -0,0 +1,20 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    {
+      "vc": "http://www.w3.org/2018/credentials#",
+      "jwt": "http://example.com/jwt#"
+    }
+  ],
+  "@type": "JwtVerifiableCredential",
+  "vc:credentialSubject": {
+    "@type": "JwtCredentialSubject",
+    "jwt:iss": {"@id": "vc:issuer"},
```

Comment:
this stuff needs to be in the `@context`, but I think the intention is clear.

Reply:
made some adjustments - still working through a double check on n-quads bc json-ld is hard